ark/phenotyping/som_utils.py

```diff
@@ -1015,10 +1244,11 @@ def cell_consensus_cluster(base_dir, max_k=20, cap=3, column_prefix='cluster',
             print(output.strip())
 
 
-def visualize_cell_cluster_data(base_dir, cluster_name, column_prefix, cell_cluster_col='cluster',
-                                dpi=None, center_val=None, overlay_values=False, colormap="vlag",
-                                save_dir=None, save_file=None):
-    """For cell-level analysis, visualize the average cluster results for each cluster
+def visualize_avg_p2c_counts(base_dir, cluster_name, column_prefix, cell_cluster_col='cluster',
```

Comment:
Does this need to be a function?

Reply:
@ngreenwald so there are a few more computational steps done in this function before passing it on to `draw_heatmap`. Similar to `visualize_cell_cluster_channel_avg`, I can see if I can remove some of this preprocessing to a separate function (or other functions).